test/ctes_in_my_pg_test.rb

```diff
@@ -22,6 +22,74 @@
       _(query.to_sql).must_match(/WITH RECURSIVE "lucky_number_seven" AS \(SELECT "people".* FROM "people"(\s+)WHERE "people"."lucky_number" = 7\) SELECT "people".* FROM "people" JOIN lucky_number_seven ON lucky_number_seven.id = people.id/)
     end
 
+    describe 'PostgreSQL server supports materialization specifiers' do
+      it 'generates an expression with materialized' do
```

Comment:
I didn't have PG 12 running on my machine, so these tests failed initially.  But it's also good to test `supports_materialization_specifiers?` w/o stubbing it, so I dunno.  I upgraded to PG 12, and all passed.